README.md

```diff
@@ -487,27 +769,151 @@ dlv attach --continue --accept-multiclient --api-version 2 --headless --listen :
 5. Use your IDE's debug capabilities to attach to `localhost:40000` to start your debug session. This will vary based
    on which IDE is being used.
 
-## Request Examples
+## <a name='RequestExamples'></a>Request Examples
+
+### <a name='QuerytheMetadataserver'></a>Query the Metadata server
+
+> :warning: Double check you already applied the proper permissions to access the O2IMS API and created the test-client service account. See section [Testing API endpoints on a cluster](#testing-API-endpoints-on-a-cluster)
+
 
-### Infrastructure Inventory Subscription (Resource Server)
+Notice the `API_URI` is the route HOST/PORT column of the oran-o2ims operator. 
 
-#### GET Infrastructure Inventory Subscription List
+```sh
+$ oc get routes -n oran-o2ims
+NAME        HOST/PORT                                                   PATH                                                   SERVICES                    PORT   TERMINATION          WILDCARD
+api-2fv99   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /                                                      metadata-server             api    reencrypt/Redirect   None
+api-cbvbz   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /o2ims-infrastructureInventory/v1/resourceTypes        resource-server             api    reencrypt/Redirect   None
+api-v92jb   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /o2ims-infrastructureInventory/v1/deploymentManagers   deployment-manager-server   api    reencrypt/Redirect   None
+api-xllml   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /o2ims-infrastructureInventory/v1/resourcePools        resource-server             api    reencrypt/Redirect   None
+```
+
+Export the o2ims endpoint as the API_URI variable so it can be re-used in the requests.
+
+```sh
+export API_URI=o2ims.apps.${DOMAIN}
+```
+
+#### <a name='GETapi_versions'></a>GET api_versions
+
+To get the api versions supported
+
+```sh
+$ curl --insecure --silent --header "Authorization: Bearer $MY_TOKEN" 
```

Comment:
We should be consistent with the use of brackets around variables.  `${MY_TOKEN}` rather than `$MY_TOKEN`